actions/get-devbuild-comment/action.yaml

```diff
@@ -0,0 +1,49 @@
+name: "Get devbuild comment"
+description: "Gets a comment with description for how to use overrides for devbuilds"
+author: "niek.vanstaveren@qlik.com"
+
+inputs:
+  package:
+    description: "package name"
+    required: true
+  namespace:
+    description: "qmfe namespace"
+    required: true
+  version:
+    description: "qmfe component version"
+    required: true
+  file:
+    description: "Systemjs file to overwrite"
+    required: true
+
+outputs:
+  comment:
+    description: "comment with instructions for how to use devbuild overrides"
+    value: ${{ steps.comment.outputs.comment }}
+
+runs:
+  using: composite
+  steps:
+    - name: Get qmfe overrides
+      id: comment
+      env:
+        PACKAGE: ${{ inputs.package }}
+        NAMESPACE: ${{ inputs.namespace }}
+        VERSION: ${{ inputs.version }}
+        FILE: ${{ inputs.file }}
+      run: |
+        COMMENT="1. Open Browser DevTools (F12)"$'\n'
+        COMMENT+="2. Go to Console tab"$'\n'
+        COMMENT+="3. Copy & Paste:"$'\n'
+        COMMENT+="- localStorage.setItem('import-map-override:$PACKAGE', 'https://d19uocc72ax6zk.cloudfront.net/$NAMESPACE/$PACKAGE/$VERSION/$FILE')"$'\n'
+        COMMENT+="4. Refresh the page (F5)"$'\n'
+        COMMENT+="5. ☕️"$'\n'
+
+        # Set the output for the GitHub Actions step using the 'EOF' syntax
```

Comment:
Include a timestamp when the build was done and what build number we used

Reply:
Not sure how the timestamp will help. 
What build number do you want? Maybe the commit hash to know to which commit it related? Could even try to get a link to the commit

Reply:
Since we are updating an existing comment I just wanted something in the text to show what build that has successfully been published. A link to the workflow run would also be suitable

Reply:
Okay, I added the actual commit in the message. It shows the link to the commit see here https://github.com/qlik-trial/sn-org-chart/pull/3#issuecomment-2109523634